pkgs/build-support/build-dotnet-module/default.nix

```diff
@@ -60,6 +71,7 @@ let
       inherit sha256;
     };
   });
+  _localDeps = linkFarmFromDrvs "${name}-local-deps" projectReferences;
```

Comment:
Maybe it makes sense to assert that the projectReferences are configured with `packNupkg = true`. I guess this attribute is visible in `.drvAttrs` or somewhere?

Reply:
I was thinking of that, but then we restrict the usage of `projectReferences` variable for propagating arbitrary `nupkg` dependencies, which might be a handy usecase for private artifacts.

Reply:
This removes the possibility of abusing `projectReferences` attribute for private `nupkg` resources which is a handy side-effect.